app/helpers.php

```diff
@@ -269,19 +269,19 @@ function get_metadata($entry)
         $entry = $entry->fields;
     }
 
-    $image = $entry->metadata->fields->image;
+    $image = $entry->metadata ? $entry->metadata->fields->image : null;
 
     if (! $image && data_get($entry, 'coverImage')) {
-        $image = $entry->coverImage->url.'?w=1200&h=1200&fm=jpg&fit=fill';
+        $image = $entry->coverImage->url;
     }
 
     $data = [
-        'title' => $entry->metadata->fields->title ?: $entry->title,
+        'title' => $entry->metadata ? $entry->metadata->fields->title : $entry->title,
```

Comment:
Would using `data_get` be easier then these ternaries?
```suggestion
        'title' => data_get($entry, 'metadata.fields.title, $entry->title)
```

Reply:
(small fry - I just really wanted to use that new suggestion thingy!)

Reply:
haha. Yeah, I was debating that... 🤔